modules/ROOT/pages/deploy-on-premises.adoc

```diff
@@ -213,6 +215,33 @@ include::example$mmp-concept-config.xml[]
 mvn clean package deploy -DmuleDeploy
 ----
 
+== Deploy Applications in Parallel
+
+You can deploy applications in parallel to on-premises Mule instances. Deploying applications in parallel reduces the startup time when you deploy a large number of apps. To prevent deployment failure, ensure that applications are not dependent on each other because a particular start order cannot be guaranteed.
+
+To enable parallel deployment:
+
+. Package or export your Mule applications to the `/apps` directory of your Mule runtime engine instance.
+. Start Mule using the `-M-Dmule.deployment.parallel` option:
++
+```
+mule -M-Dmule.deployment.parallel
+```
+
+After performing these steps, all applications in the `/apps` directory start in parallel.
+
+== Update Applications at Runtime
+
+Updating a Mule application at runtime can be a complex change involving class modifications, endpoint modifications (for example, changing ports, etc), as well as reconfiguring flows. As a result, any application update does a graceful app shutdown under the hood and reconfigures itself. In practice, this is pretty transparent to the user and happens within seconds.
+
+There are several ways you can update an application:
+
+* By dropping the modifications over an existing exploded app folder and touching the master configuration file (`mule-config.xml` located in the app's root directory by default).
+* By dropping a new `jar` version of the app into the `$MULE_HOME/apps` directory. Mule detects the `jar` as an existing application and ensures the updated by a clean redeployment of the app. +
+Note that Mule discards any modifications to the old application folder. The new app folder is a clean exploded application from a `jar`.
```

Comment:
Again, not certain about "clean exploded application."
Are we saying that adding the new JAR file (folder?) replaces the existing (old) folder, including any changes to that old folder?
Again, suggest having a peer writer take a look at the wording, since it is so new to me.